index.html

```diff
@@ -1,170 +1,203 @@
+      </div>
+      <div class="project">
+        <img class="pr-img" src="./assets/pr6.png">
+        <div class="project-ins">
+          <h3>Project name goes here</h3>
+          <ul>
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+          <a href="#">
+            <button type="button">
+              <span>See this project</span>
+              <img class="pr-button-img" src="./assets/Union.png">
+            </button>
+          </a>
         </div>
-       </div>
-    </section>
-    <section class="about">
-        <h2>About</h2>
-        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+      </div>
+    </div>
+  </section>
+  <section class="about">
+    <h2>About</h2>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+      magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+    <button type="button">
+      Get my CV
+      <img src="./assets/ic_download.svg">
+    </button>
+    <div class="languages">
+      <h4 class="set-h">Languages</h4>
+      <ul class="set-ul">
+        <li>JavaScript(ES5/ES6)</li>
+        <li>Ruby</li>
+        <li>HTML</li>
+        <li>CSS</li>
+      </ul>
+      <img class="set-img" src="./assets/hash.png">
+    </div>
+    <div class="frameworks">
+      <h4 class="set-h">Frameworks</h4>
+      <ul class="set-ul">
+        <li>Bootstrap</li>
+        <li>Ruby on rails</li>
+        <li>RSpec</li>
+        <li>Capibara</li>
+        <li>Selenium</li>
+      </ul>
+      <img class="set-img" src="./assets/Group 18.png">
+    </div>
+    <div class="skills">
+      <h4 class="set-h">Skills</h4>
+      <ul class="set-ul">
+        <li>Codekit</li>
+        <li>Codepan</li>
+        <li>GitHub</li>
+        <li>GitLab</li>
+        <li>Terminal</li>
+      </ul>
+      <img class="set-img" src="./assets/ic_skills.png">
+    </div>
+  </section>
+  <section class="contact-form">
+    <h2>Get started</h2>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+    <form method="post" action="https://formspree.io/f/xvoyvapl">
+        <input class="form-items" type="text" name="name" id="full-name" maxlength="30" placeholder="Your name" required>
+        <input class="form-items" type="email" name="email" id="email" placeholder="Your Email" required>
+        <textarea class="form-items" name="text-area" id="text-area" maxlength="500" cols="30" rows="6" placeholder="Write your message here" required></textarea>
+        <a href="#">
+            <button type="submit" class="start-collab">Start collaboration</button>
+        </a>
+    </form>
+  </section>
```

Comment:
- [ ] Good job so far your website looks awesome 👏 , but to match Figma design kindly remove the space between the footer and the form section and instead add some margins to the forms 🙏 , here's an example : 


```CSS
form {
    display: flex;
    flex-direction: column;
    /* Add margin-bottom to the form */
    margin-bottom: 50px;
}

footer {
    width: 100%;
   /* Remove the margin-top: 64px; for the footer */
    background-color: #1a2236;
}
```


Before           | After
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/89282221/199257175-27c4e06d-990c-4da9-b70a-100ae9797106.png) | ![image](https://user-images.githubusercontent.com/89282221/199258010-422e12f1-ecc6-42df-ba41-18e122d19161.png)